domain/src/main/java/com/architect/codes/mu8/events/EventsUseCase.kt

```diff
@@ -0,0 +1,9 @@
+package com.architect.codes.mu8.events
+
+interface EventsUseCase {
+    suspend fun getEvents(): List<Event>
+}
+
+class EventsUserCaseImpl(private val eventsRepository: EventsRepository) : EventsUseCase {
+    override suspend fun getEvents(): List<Event> = eventsRepository()
```

Comment:
You can use here `operator invoke()` too since there is only one method to call.

Reply:
Done